Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift

```diff
@@ -99,19 +99,16 @@ internal final class ChangeEmailViewController: UIViewController {
       |> \.text %~ { _ in Strings.New_email() }
 
     _ = self.newEmailTextField
-      |> formFieldStyle
+      |> emailFieldAutoFillStyle
       |> \.returnKeyType .~ .next
       |> \.textAlignment .~ .right
-      |> \.placeholder %~ { _ in
```

Comment:
Duplicate (used in `emailFieldAutoFillStyle` who's inheriting it from `formFieldStyle`